toqito/matrix_ops/partial_trace.py

```diff
@@ -7,6 +7,8 @@
 from toqito.helper import expr_as_np_array, np_array_as_expr
 from toqito.perms import permute_systems
 
+# from toqito.channels import kraus_to_choi
+
```

Comment:
why is this commented out

Reply:
its for Modules like is_ppt.py (in state_props), channel_fidelity.py (in channel_metrics), and any modules in channel_ops can now import partial_trace directly from toqito.matrix_ops, allows toqito.channels.__init__.py to more safely import from toqito.channel_ops (like kraus_to_choi) because the dependency chain of kraus_to_choi will no longer attempt to call back into a partially initialized toqito.channels to get partial_trace.